src/views/AppStoreApp.vue

```diff
@@ -110,6 +115,11 @@
         class="app-gallery-screen mr-3"
         :key="image"
         :src="`https://getumbrel.github.io/umbrel-apps-gallery/${app.id}/${image}`"
+        @click="
+          zoom(
+            `https://getumbrel.github.io/umbrel-apps-gallery/${app.id}/${image}`
+          )
+        "
```

Comment:
In my opinion (I don't actually know vue, not sure if this works), using this.src instead of defining the URL again would make it easier:

```suggestion
        @click="
          zoom(
            this.src
          )
        "
```

But wait for a review from another developer, I'm not sure if this is actually better.

Reply:
Can't work like that because src doesn't exist in the component it's a binding for the img

Reply:
Thank you, sorry for the useless review then.

Reply:
No worries :wink: